### PR TITLE
Fix native image build

### PR DESCRIPTION
**Pull request by imkiva**
bors merge

### CODE REVIEW

**Comment by imkiva** [comment]
bors cancel

**Comment by imkiva** [comment]
bors merge

File: buildSrc/src/main/groovy/org/aya/gradle/CommonTasks.groovy [hunk 1]
```diff
@@ -54,6 +54,7 @@ final class CommonTasks {
     var task = project.tasks.register('generateNativeImageConfig', GenerateReflectionConfigTask) {
       outputDir = configGenDir
       inputFile = configTemplateFile
+      outputs.upToDateWhen { false }
```
Comment (ice1000) [inline buildSrc/src/main/groovy/org/aya/gradle/CommonTasks.groovy:57]: Why

Comment (imkiva) [reply buildSrc/src/main/groovy/org/aya/gradle/CommonTasks.groovy:57]: It keeps showing up-to-date here. So I googled "gradle task always rerun" 😭. Do you have better idea?

Comment (ice1000) [reply buildSrc/src/main/groovy/org/aya/gradle/CommonTasks.groovy:57]: > It keeps showing up-to-date here. So I googled "gradle task always rerun" 😭. Do you have better idea?

I'll see what I can do

Comment (ice1000) [reply buildSrc/src/main/groovy/org/aya/gradle/CommonTasks.groovy:57]: I don't understand...

Comment (ice1000) [reply buildSrc/src/main/groovy/org/aya/gradle/CommonTasks.groovy:57]: I don't think there's anything I can do. Let's keep your changes

**Pull request merged**